### ISSUE TITLE
Adding jq to base image

### ISSUE THREAD

**Issue by xrl** [elastic/elasticsearch-docker#211]
## Feature Description

I am doing some custom bootstrapping code using `elasticsearch:6.4.1` and I end up adding the jq package every time. Could we put that in the standard distribution? It'll be useful for anyone who uses curl and wants to extract specific values.

The `jq` package has 1 dependency, oniguruma. That's it.

**Comment by drewr** [comment]
I agree with @jasontedor.  Leave the base image with the essentials necessary for only running Elasticsearch.  You can `curl | jq` in whatever environment you're testing from.

If you really do need to do it from the container, this is where Docker shines anyway. Make your own container:

```
FROM docker.elastic.co/elasticsearch/elasticsearch:6.4.2
RUN yum install --y jq
[...]
```

**Comment by xrl** [comment]
I am trying to keep the Elasticsearch containers are "untouched" as possible so I'm always able to grab images directly from the docker.elastic.co repo.

I have set up a script which can generate CSRs using the elasticsearch-certutil script (which uses unzip, yum installed in the Elasticsearch Dockerfile), uses curl and jq to talk to our cert signing services, and configured everything so x-pack is ready to go on boot.

I may consider vendoring jq/oniguruma RPMs to S3 and downloading them directly without doing traffic/time heavy yum operations.

**Comment by xrl** [comment]
I am OK with the reasoning behind keeping JQ out of the base image. The line does have to be drawn at some point. `zip` OK, `jq`, not OK! 😄
